CHANGELOG.md

```diff
@@ -2,9 +2,9 @@
 
 - **[BREAKING CHANGE]** Use `title` instead of `children` prop in `Stepper`
 - **[NEW]** Add `StepperOverlay` component
-- **[NEW]** Add `StepperFullScreen` component
+- **[NEW]** Add `StepperSection` component
```

Comment:
I don't like the `Section` suffix: we're using it for large layout things. @grosbouddha WDYT?

Reply:
This naming solution was proposed by @grosbouddha in [previous PR](https://github.com/blablacar/ui-library/pull/586)

Reply:
Yep, I agree with @mcellier. All sections should take care of their containing page vertical layout ie being a centered column (it can be as simple as wrapping them in a BaseSection).
You could as well not call them *Section but I believe I would prefer to have AutoCompleteSection and StepperSection with the proper section layout.
@wakooka wdyt ?

Reply:
Stepper and DatePicker are wrapped by `section` component
AutoComplete was not but I fixed it in [following PR](https://github.com/blablacar/ui-library/pull/590)